nextflow.config

```diff
@@ -1,150 +1,253 @@
+        shifter.enabled        = false
+        charliecloud.enabled   = false
+    }
+    shifter {
+        shifter.enabled        = true
+        docker.enabled         = false
+        singularity.enabled    = false
+        podman.enabled         = false
+        charliecloud.enabled   = false
+    }
+    charliecloud {
+        charliecloud.enabled   = true
+        docker.enabled         = false
+        singularity.enabled    = false
+        podman.enabled         = false
+        shifter.enabled        = false
+    }
+    gitpod {
+        executor.name          = 'local'
+        executor.cpus          = 16
+        executor.memory        = 60.GB
+    }
+    test        { includeConfig 'conf/test.config'        }
+    test_grocap { includeConfig 'conf/test_grocap.config' }
+    test_copro  { includeConfig 'conf/test_copro.config'  }
+    test_full   { includeConfig 'conf/test_full.config'   }
 }
 
+
 // Load igenomes.config if required
-if(!params.igenomesIgnore){
-  includeConfig 'conf/igenomes.config'
+if (!params.igenomes_ignore) {
+    includeConfig 'conf/igenomes.config'
+} else {
+    params.genomes = [:]
+}
+
+
+// Export these variables to prevent local Python/R libraries from conflicting with those in the container
+// The JULIA depot path has been adjusted to a fixed path `/usr/local/share/julia` that needs to be used for packages in the container.
+// See https://apeltzer.github.io/post/03-julia-lang-nextflow/ for details on that. Once we have a common agreement on where to keep Julia packages, this is adjustable.
+
+env {
+    PYTHONNOUSERSITE = 1
+    R_PROFILE_USER   = "/.Rprofile"
+    R_ENVIRON_USER   = "/.Renviron"
+    JULIA_DEPOT_PATH = "/usr/local/share/julia"
 }
 
 // Capture exit codes from upstream processes when piping
 process.shell = ['/bin/bash', '-euo', 'pipefail']
 
+def trace_timestamp = new java.util.Date().format( 'yyyy-MM-dd_HH-mm-ss')
 timeline {
-  enabled = true
-  file = "${params.tracedir}/execution_timeline.html"
+    enabled = true
+    file    = "${params.tracedir}/execution_timeline_${trace_timestamp}.html"
 }
 report {
-  enabled = true
-  file = "${params.tracedir}/execution_report.html"
+    enabled = true
+    file    = "${params.tracedir}/execution_report_${trace_timestamp}.html"
 }
 trace {
-  enabled = true
-  file = "${params.tracedir}/execution_trace.txt"
+    enabled = true
+    file    = "${params.tracedir}/execution_trace_${trace_timestamp}.txt"
 }
 dag {
-  enabled = true
-  file = "${params.tracedir}/pipeline_dag.svg"
+    enabled = true
+    file    = "${params.tracedir}/pipeline_dag_${trace_timestamp}.html"
 }
 
 manifest {
-  name = 'nf-core/nascent'
-  author = 'Ignacio Tripodi, Margaret Gruca'
-  homePage = 'https://github.com/nf-core/nascent'
-  description = 'Nascent Transcription Processing Pipeline'
-  mainScript = 'main.nf'
-  nextflowVersion = '>=0.32.0'
-  version = '1.0'
+    name            = 'nf-core/nascent'
+    author          = 'Edmund Miller'
```

Comment:
same here about the authors, would suggest keeping everyone that was there (again unless you guys agreed on something else :) )